server/src/main/java/org/spine3/server/storage/memory/InMemoryCommandStorage.java

```diff
@@ -104,4 +102,10 @@ private void put(CommandId id, CommandStorageRecord record) {
     private CommandStorageRecord get(CommandId id) {
         return storage.get(id);
     }
+
+    private void checkNotContains(CommandId id, String commandId) {
+        if (storage.containsKey(id)) {
```

Comment:
We can add an `update` method, but it is needed only in EntityStorage now. Also, we don't know what method to use: `update` or `insert`, so the `contains` method is also needed (only for entity storage). And implementing these methods in all storages violates the YAGNI principle.
Or we can document `write` method in entity storage that it can also update the record, and that it only inserts one record in other storages.